src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -15,6 +15,7 @@
 import numpy as np
 import torch
 from habitat.core.agent import Agent
+from habitat.core.spaces import EmptySpace
```

Comment:
if we are adding habitat as a dependency we should update install insturctions accordingly

Reply:
added to `projects/stretch_ovmm/README.md`